mmv1/products/firebase/ProjectLocation.yaml

```
@@ -37,6 +37,12 @@ references: !ruby/object:Api::Resource::ReferenceLinks
 import_format: ['projects/{{project}}', '{{project}}']
 skip_delete: true
 skip_sweeper: true
+docs:
```

Looks like we need to specify the "type" of this.

```suggestion
docs: !ruby/object:Provider::Terraform::Docs
```

---

updated docs in https://github.com/GoogleCloudPlatform/magic-modules/pull/8663